lib/libutils/isoc/snprintf.c

```diff
@@ -200,6 +227,9 @@ kprintf(const char *fmt0, int oflags, void *vp, char *sbuf, va_list ap)
 	const char *xdigs;	/* digits for [xX] conversion */
 	char bf[KPRINTF_BUFSIZE]; /* space for %c, %[diouxX] */
 	char *tailp;		/* tail pointer for snprintf */
+#ifdef __KERNEL__
+	char uuidb[UUID_STRING_SIZE];
```

Comment:
To conserve stack usage please use `bf` above instead as `uuidb` and `bf` never are used at the same time and actually serve similar purposes.